panda:
  ros__parameters:
    control_dt: 0.5 # 2 Hz
    joint_controller_name: 'joint_group_position_controller'
    joint_control_topic: 'joint_group_position_controller/commands'
    end_effector_target_topic: 'end_effector_target_pose'
    end_effector_pose_topic: 'end_effector_pose'
    model_file: 'description/models/panda/panda.urdf'
    base_frame: 'panda_link0'
    end_effector_frame: 'end_effector_frame'
    finger_joint_tag: 'finger' # tag for the joints to exclude from the IK optimization
    arm_joint_tag: 'panda_joint'
    initial_joint_angles: [0., -0.785, 0., -2.356, 0., 1.571, 0.785, 0., 0.]
    joint_remapping: # necessary because ros messes up the joint states ordering
      panda_joint1:         1
      panda_joint2:         2
      panda_joint3:         4
      panda_joint4:         5
      panda_joint5:         6
      panda_joint6:         7
      panda_joint7:         8
      panda_finger_joint1:  0
      panda_finger_joint2:  3